src/javascript/directives/angucomplete-alt.js

```diff
@@ -57,7 +57,7 @@ angular.module('angucomplete-alt').directive('angucompleteAlt', ['$q', '$parse',
           return target;
         }
 
-        if (target.className.indexOf($scope.matchClass) >= 0) {
+        if (typeof target.className === 'string' && target.className.indexOf($scope.matchClass) >= 0) {
```

Comment:
this is fine, though i don't think className would ever be a non-string value other than `undefined`

IMO this would be marginally better if it were `if (angular.isDefined(target.className) && target...)`